### DISCUSSION TITLE
Failed interpolation after downloading GFS05_ensemble --second-flush

### DISCUSSION THREAD

**Discussion by magheralex** [open-meteo/open-meteo#1056] [General]
Hi Patrick,
I've tried to download the second flush for GFS05_ens using:
docker run -it --rm -v open-meteo-data:/app/data ghcr.io/open-meteo/open-meteo download-gfs gfs05_ens --only-variables temperature_2m,cloud_cover,precipitation,frozen_precipitation_percent,showers,cape,wind_gusts_10m,categorical_freezing_rain,visibility,lifted_index --second-flush --run 2024100400

and got the following error:

[ INFO ] Downloading file https://noaa-gefs-pds.s3.amazonaws.com/gefs.20241003/0                                                                       0/atmos/pgrb2bp5/gep30.t00z.pgrb2b.0p50.f840 [range 57022559-57322718......]
[ INFO ] Finished downloading 2.87 GB in 72m
[ INFO ] [ Convert temperature_2m (31 nMembers) 2024-10-03T00:00 to 2024-11-07T0
0:00 (3-hourly) ] Starting
App/InterpolationInplace.swift:40: Fatal error: First 6 timesteps only contains NaN, followed by valid data. This leads to issues in interpolation.

💣 Program crashed: Illegal instruction at 0x00007f83d2f5ed12

Thread 3 crashed:

 0 0x00007f83d2f5ed12 _assertionFailure(_:_:file:line:flags:) + 354 in libswiftC                                                                       ore.so
 1 Array<A>.interpolateInplace(type:skipFirst:time:grid:locationRange:) + 2529 i                                                                       n openmeteo-api at /build/Sources/App/Helper/InterpolationInplace.swift:40:13
 2 Array3DFastTime.interpolateInplace(type:skipFirst:time:grid:locationRange:) i                                                                       n openmeteo-api at Sources/App/Helper/InterpolationInplace.swift:12:14
 3 specialized closure #6 in static GenericVariableHandle.convert(logger:domain:                                                                       createNetcdf:run:handles:) + 1900 in openmeteo-api at /build/Sources/App/Helper/                                                                       Writer/GenericVariableHandle.swift:115:24
 4 OmFileSplitter.updateFromTimeOrientedStreaming(variable:time:skipFirst:scalef                                                                       actor:compression:storePreviousForecast:supplyChunk:) + 548 in openmeteo-api at                                                                        /build/Sources/App/Helper/OmFile.swift:314:28
 5 specialized static GenericVariableHandle.convert(logger:domain:createNetcdf:r                                                                       un:handles:) + 7183 in openmeteo-api at /build/Sources/App/Helper/Writer/Generic                                                                       VariableHandle.swift:91:20
 6 static GenericVariableHandle.convert(logger:domain:createNetcdf:run:handles:c                                                                       oncurrent:) + 226 in openmeteo-api at /build/Sources/App/Helper/Writer/GenericVa                                                                       riableHandle.swift:35:17
 7 specialized GfsDownload.downloadRun(using:signature:run:domain:) in openmeteo                                                                       -api at /build/Sources/App/Gfs/GfsDownload.swift:121
 8 specialized GfsDownload.run(using:signature:) in openmeteo-api at /build/Sour                                                                       ces/App/Gfs/GfsDownload.swift:55
 9 AsyncCommand.run(using:) in openmeteo-api at /build/.build/checkouts/console-                                                                       kit/Sources/ConsoleKitCommands/Async/AsyncCommand.swift:94
10 specialized AsyncCommandGroup.run(using:) in openmeteo-api at /build/.build/c                                                                       heckouts/console-kit/Sources/ConsoleKitCommands/Async/AsyncCommandGroup.swift:30
11 Console.run(_:with:) in openmeteo-api at /build/.build/checkouts/console-kit/                                                                       Sources/ConsoleKitCommands/Base/Console+Run.swift:91
12 Application.startup() in openmeteo-api at /build/.build/checkouts/vapor/Sourc                                                                       es/Vapor/Application.swift:232
13 Application.execute() in openmeteo-api at /build/.build/checkouts/vapor/Sourc                                                                       es/Vapor/Application.swift:194
14 0x00007f83d2dd90b0 completeTaskAndRelease(swift::AsyncContext*, swift::SwiftE                                                                       rror*) in libswift_Concurrency.so

Backtrace took 40.77s

Do you have any idea what happened? Thanks in advance!

**Comment by patrick-zippenfenig** [comment]
Hi,
this could be a false alarm. Are you using the latest docker image version? I thought I removed most of those issues in the last months.

**Comment by magheralex** [comment]
Hi,
I was running 1.1.13. I updated to latest version and now it works. Thank you for your prompt answer.
